packages/synapse-react-client/src/components/ExperimentalMode/ExperimentalMode.tsx

```diff
@@ -35,16 +35,15 @@ const ExperimentalMode: React.FunctionComponent<ExperimentalModeProps> = ({
   }, [])
 
   const createExperimentalModeCookie = () => {
-    cookies.set(EXPERIMENTAL_MODE_COOKIE, { path: '/' })
+    cookies.set(EXPERIMENTAL_MODE_COOKIE, 'true', { path: '/' })
     setIsExperimentalModeOn(true)
     if (onExperimentalModeToggle) {
       onExperimentalModeToggle(true)
     }
   }
 
   const deleteExperimentalModeCookie = () => {
-    // cookie.remove requires to re-calculate date obj each time, this is more straightforward
```

Comment:
Maybe this comment was true at some point, but it certainly isn't now.  Just use cookies.remove() (I suspect that this will fix the bug)